pages/finance.tsx

```diff
@@ -21,12 +21,18 @@ function Finance() {
     startDate: dateBeforeDays(3),
     endDate: dateAfterDays(4),
   })
+  const [activeCell, setActiveCell] = useState([0, 0])
+  function getActiveCell(cell: any) {
+    setActiveCell(cell)
```

Comment:
why not send setActiveCell to all components directly instead of wrapping with getActiveCell? The operation is more of setting the value